Source/WebCore/css/html.css

```diff
@@ -790,28 +790,34 @@ input:-webkit-autofill, input:-webkit-autofill-strong-password, input:-webkit-au
 input:is([type="radio"], [type="checkbox"]) {
     margin: 3px 2px;
     box-sizing: border-box;
-    border: initial;
+    border: -internal-auto-base(initial, 0.3em solid currentColor);
```

Comment:
Doesn't match [the spec](https://drafts.csswg.org/css-forms/#basic-appearance-stylesheet). Does the PR need updating or the spec need updating or is there no decision here yet (in which case, do we have an issue filed)?

Reply:
This was removed.